tracing-flame/tests/collapsed.rs

```diff
@@ -0,0 +1,38 @@
+use std::thread::sleep;
+use std::time::Duration;
+use tracing::{span, Level};
+use tracing_flame::FlameLayer;
+use tracing_subscriber::{prelude::*, registry::Registry};
+
+#[test]
+fn capture_supported() {
+    {
+        let (flame_layer, _guard) = FlameLayer::with_file("./tracing.folded").unwrap();
```

Comment:
I'd like to have `tempdir` pulled in for tests to handle cleanup: https://docs.rs/tempdir/0.3.7/tempdir/. Once you add that in, could you add a comment with the current destination?